OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/AnalyticsRegistry.java

```diff
@@ -6,6 +6,7 @@
 import com.google.inject.Singleton;
 
 import org.edx.mobile.util.images.ShareUtils;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Can we use `NonNull` annotation as it already been using in this class

Reply:
done.